Server/MirObjects/PlayerObject.cs

```diff
@@ -11291,6 +11292,12 @@ public void UseItem(ulong id)
                                 AddBuff(BuffType.Exp, this, Settings.Minute * time, new Stats { [Stat.ExpRatePercent] = item.GetTotal(Stat.Luck) });
                             }
                             break;
+                        case 6: //Drop
```

Comment:
Why have you used 6? 5 is not used currently.

Reply:
The shape of the drop buff item I used before is 6, now it has been changed to 5.